test/spec/module-resolution/node_modules/sub-module/index.js

```diff
@@ -0,0 +1 @@
+exports = require('./weak-map');
```

Comment:
When exporting a single object as the module, you have to use `module.exports` instead. While assigning to `exports.someExport` is fine, assigning `exports` to a new value just re-assigns the `exports` parameter within the module factory, but will have no effect on the module's exported content.
```suggestion
module.exports = require('./weak-map');
```